.github/workflows/ci.yml

```diff
@@ -68,23 +62,19 @@ jobs:
           java-version: "11"
```

Comment:
Maybe it would be good to create a new env variable for java version?

Reply:
Normally yes, in this case not really IMO. It would be better to keep it as-is so it's as similar as possible to the what's on e.g.: `release/7.2.1` so that if the need to cherry-pick any more fixes this can be done without getting a headache + this branch will never move from java-version = 11.